dartagnan/src/main/java/com/dat3m/dartagnan/program/analysis/alias/AndersenAliasAnalysis.java

```diff
@@ -89,6 +89,7 @@ private void run(Program program) {
         for (MemoryCoreEvent e : memEvents) {
             processLocs(e);
         }
+        //FIXME: Add handling for thread parameters (or get rid of this class)
```

Comment:
@ThomasHaas you mentioned to me the other day some bug related to passing arguments and alias analysis ... how was this solved?

Reply:
Can we make use of the same fix? i.e. is this an easy fix?

While we normally do not use this option, I would prefer to keep it in case we eventually need it (assuming maintaining the extra cost is not too much burden)

Reply:
At first glance, the code is quite different and I don't know how to fix it easily. Maybe René knows the code better and can tell, but from my perspective the fix is not trivial.